AppController/djinn.rb

```diff
@@ -6324,15 +6184,23 @@ def get_all_stats(secret)
       }
     end
 
-    all_stats["public_ip"] = my_node.public_ip
-    all_stats["private_ip"] = my_node.private_ip
-    all_stats["roles"] = my_node.jobs or ["none"]
-    Djinn.log_debug("All stats: #{all_stats}")
+    node_stats["cloud"] = my_node.cloud
```

Comment:
Does this value identify which cloud the node is in? We may wish to deprecate this value until we can properly support multi-cloud deployments.